atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/InternalSchemaConfig.java

```diff
@@ -41,6 +42,16 @@ public abstract class InternalSchemaConfig {
      */
     public abstract Optional<Integer> targetTransactionsSchemaVersion();
 
+    /**
+     * If specified, accepts a read of a STAGING value as COMMITTED when reading a transactions3 value.
+     * This addresses a problem faced at stacks where users have high read throughput.
+     * Eventually, this should be changed to perform some kind of locking.
+     */
+    @JsonProperty("acceptStagingReadsOnVersionThreeDangerousIKnowWhatIAmDoing")
+    public boolean acceptStagingReadsOnVersionThree() {
```

Comment:
Does this need to be annotated as a default value?

```suggestion
    @Value.Default
    public boolean acceptStagingReadsOnVersionThree() {
```

Reply:
phew! yeah that's a great catch